### PR TITLE
Stress-Test Gen

### PR DESCRIPTION
**Pull request by CodaFi**
- Adds more tests to Gen
- `@exported` XCTest so everyone can get rid of that stupid import.

### CODE REVIEW

File: SwiftCheck/TestOperators.swift [hunk 1]
```diff
@@ -117,4 +117,4 @@ public func ^||^(p1 : Testable, p2 : Testable) -> Property {
 	return disjoin(p1.property, p2.property)
 }
 
-import func XCTest.XCTFail
+@exported import XCTest
```
Comment (pthariensflame) [inline SwiftCheck/TestOperators.swift]: Whoa, whoa, whoa!  What's this annotation?

Can we use this to simplify the whole dependency-diamond thing within TypeLift?

Comment (CodaFi) [inline SwiftCheck/TestOperators.swift]: Probably.  It lets you export another framework's symbols as though they were yours.

Comment (CodaFi) [inline SwiftCheck/TestOperators.swift]: It's also undocumented, so I'm not entirely sure what else it does or if this is what it's supposed to do. But hey, it works here.

Comment (CodaFi) [inline SwiftCheck/TestOperators.swift]: OK, this thing seriously breaks code completion.  But it works so well...

**Comment by CodaFi** [comment]
For now, we're all good.  I hope @exported stabilizes soon.

:boat:

**Comment by pthariensflame** [comment]
@CodaFi I hope so too!

**Comment by robrix** [comment]
TIL

**Pull request merged**